themes/src/main/resources-community/theme/base/admin/messages/admin-messages_de.properties

```diff
@@ -0,0 +1,1513 @@
+
+manage=Verwalten
+authentication=Authentifikation
+#user-federation=User Federation
+#user-storage=User Storage
+events=Ereignisse
+realm-settings=Realm Einstellungen
+configure=Konfigurieren
+select-realm=Realm ausw\u00E4hlen
+add=Hinzuf\u00FCgen
+
+#client-storage=Client Storage
+#no-client-storage-providers-configured=No client storage providers configured
+#client-stores.tooltip=Keycloak can retrieve clients and their details from external stores.
+
+#client-scope.name.tooltip=Name of the client scope. Must be unique in the realm. Name shouldn't contain space characters as it's used as value of scope parameter
+#client-scope.description.tooltip=Description of the client scope
+#client-scope.protocol.tooltip=Which SSO protocol configuration is being supplied by this client scope
+#client-scope.display-on-consent-screen=Display On Consent Screen
+#client-scope.display-on-consent-screen.tooltip=If on, and this client scope is added to some client with consent required, then the text specified by 'Consent Screen Text' will be displayed on consent screen. If off, then this client scope won't be displayed on consent screen
+#client-scope.consent-screen-text=Consent Screen Text
+#client-scope.consent-screen-text.tooltip=Text, which will be shown on consent screen when this client scope is added to some client with consent required. Defaults to name of client scope if it's not filled
+
+#add-user-federation-provider=Add user federation provider
+#add-user-storage-provider=Add user storage provider
+#required-settings=Required Settings
+#provider-id=Provider ID
+#console-display-name=Console Display Name
+#console-display-name.tooltip=Display name of provider when linked in admin console.
+#priority=Priority
+#priority.tooltip=Priority of provider when doing a user lookup. Lowest first.
+#user-storage.enabled.tooltip=If provider is disabled it will not be considered for queries and imported users will be disabled and read-only until the provider is enabled again.
+#sync-settings=Sync Settings
+#periodic-full-sync=Periodic Full Sync
+#periodic-full-sync.tooltip=Does periodic full synchronization of provider users to Keycloak should be enabled or not
+#full-sync-period=Full Sync Period
+#full-sync-period.tooltip=Period for full synchronization in seconds
+#periodic-changed-users-sync=Periodic Changed Users Sync
+#periodic-changed-users-sync.tooltip=Does periodic synchronization of changed or newly created provider users to Keycloak should be enabled or not
+#changed-users-sync-period=Changed Users Sync Period
+#changed-users-sync-period.tooltip=Period for synchronization of changed or newly created provider users in seconds
+#synchronize-changed-users=Synchronize changed users
+#sy
```

Comment:
"Liste von Aktionen, die der Benutzer ausführen soll, "

Reply:
"ein Zur\u00FCcksetz-E-Mail" -> "eine Zur\u00FCcksetzen-E-Mail"

Reply:
"bem Benutzer" -> "dem Benutzer"

Reply:
"E-Mail Adresse" -> "E-Mail-Adresse"

Reply:
"pers\u00F6nlichen Informationen" -> "Profil-Informationen"  ?

Reply:
Should this probably be "E-Mail zum Zurücksetzen des Passworts" instead of "Zurücksetz-Mail"?

Reply:
> Should this probably be "E-Mail zum Zurücksetzen des Passworts" instead of "Zurücksetz-Mail"?

I would agree on that. "Zurücksetz-Mail" sounds oddly strange to me.